Formula/hledger.rb

```diff
@@ -22,54 +23,23 @@ class Hledger < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "87c5cedfe3a02b5bc598c33a64d89b947e769c0c48ce2df7c9709c9311f22fb9"
   end
 
-  depends_on "ghc" => :build
+  depends_on "ghc@9" => :build
```

Comment:
I think it will build with both `ghc` (8) and `ghc@9`. The default stack.yaml is for GHC 9.2 though it will use a nightly resolver.

Reply:
Need to rebase and switch to `ghc` after #110086 which should make GHC 9.2.4 the default.